docs/docs/keymaps/behaviors/mouse-emulation.md

```diff
@@ -69,3 +75,127 @@ This example will send press of the fourth mouse button when the binding is trig
+
+```
+&mmv MOVE_LEFT
+```
+
+### Input Processors
+
+If you want to apply any [input processors](../input-processors/index.md#input-processors-overview) to `&mmv` you can do so by referencing `&mmv_input_listener`, e.g.:
+
+```dts
+&mmv_input_listener {
+    input-processors = <&zip_temp_layer 2 2000>;
+}
+```
+
+## Mouse Scroll
+
+This behavior sends vertical and horizontal scroll events to the connected host.
+
+### Behavior Binding
+
+- Reference: `&msc`
+- Parameter: A `uint32` with 16-bits each used for vertical and horizontal velocity.
+
+The following defines can be passed for the parameter:
+
+| Define       | Action       |
+| :----------- | :----------- |
+| `SCRL_UP`    | Scroll up    |
+| `SCRL_DOWN`  | Scroll down  |
+| `SCRL_LEFT`  | Scroll left  |
+| `SCRL_RIGHT` | Scroll right |
+
+Additionally, if you want to pass a different max speed than the default for the `SCRL_*` defines, custom X and Y velocity values can be passed with `MOVE_X` and `MOVE_Y`, e.g. `MOVE_X(5)` or `MOVE_Y(-5)`.
+
+### Examples
+
+The following will send a scroll down event to the host when pressed/held:
+
+```
+&msc SCRL_DOWN
+```
+
+The following will send a scroll left event to the host when pressed/held:
+
+```
+&msc SCRL_LEFT
+```
+
+:::note
+
+If you enabled [smooth scrolling](../../config/pointing.md#kconfig) then you will want to use the same `MOVE_UP`, `MOVE_DOWN`, etc values instead of the smaller `SCRL_*` parameters.
+
+:::
+
+### Input Processors
+
+If you want to apply any [input processors](../input-processors/index.md#input-processors-overview) to `&msc` you can do so by referencing `&msc_input_listener`, e.g.:
+
+```dts
+&msc_input_listener {
+    input-processors = <&zip_temp_layer 2 2000>;
+}
+```
+
+### Advanced Configuration
+
+Both `&mmv` and `&msc` are instances of the same `"zmk,behavior-input-two-axis"` behavior. As such, the following settings can be applied to either behavior, e.g.:
+
+```dts
+&mmv {
+    trigger-period-ms = <12>;
+    delay-ms = <15>;
+    time-to-max-speed-ms = <600>;
+    acceleration-exponent = <1>;
+};
+```
+
+| Property                | Description                                                                                                         |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| `trigger-period-ms`     | How many milliseconds between generated input events based on the current speed/direction.                          |
+| `delay-ms`              | How many milliseconds to delay any processing or event generation when first pressed.                               |
+| `time-to-max-speed-ms`  | How many milliseconds it takes to accelerate to the curren max speed.                                               |
+| `acceleration-exponent` | The acceleration exponent to apply: `0` - uniform speed, `1` - uniform acceleration, `2` - exponential acceleration |
```

Comment:
It'd be good to move this table under `config/pointing.md` for the `"zmk,behavior-input-two-axis"` device config, then link there. It also would need a default value column as usual.

Also need defaults documented for &mmv and &msc. Not sure exactly where, maybe we can update the example above to just show the two nodes' definitions?

Reply:
On reviewing the input processors, I noticed that those have their config defined in their own pages rather than the config page (but they aren't in table format either.) This is a bit inconsistent but I think I'd still prefer this table to be moved to a new section in the config page.

Reply:
Yeah, input processors is likely to be an expanding set, much like behaviors, so I modeled the docs for them on behaviors, and gave them their own pages. I can definitely move the two axis input into the `config/pointing.md` and reference it here though.

Reply:
@caksoylar Ok, pushed the proposed changes to document the behavior in `config/behaviors.md`
and reference it from here. Thoughts?

Reply:
That looks good, thanks. What do you think about documenting the defaults for `mmv` and `msc` specifically? I had suggested that the example block above could be repurposed to show the default values, in two separate `&mmv` and `&msc` blocks.

Reply:
Tweaked this based on the suggestion. Thoughts?

Reply:
Thanks! A couple notes on a second read:
- `s/are ase follows:/are as follows:/`
- In config page description, `2 - exponential acceleration` is mathematically incorrect, we should either call it "uniform jerk" or (better) "linear acceleration" IMO.